examples/centralized_design/variables.tf

```diff
@@ -15,6 +15,27 @@ variable "ssh_key_name" {
   type        = string
 }
 
+### IAM
+variable "iam_policies" {
+  description = "A map defining an IAM policies, roles etc."
+  type        = any
+
+  default = {
+    spoke = {
+      create_instance_profile = true
+      instance_profile_name   = "centralized_spoke_profile"
+      role_name               = "spoke_role"
+      policy_arn              = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
+    }
+    vmseries = {
+      create_instance_profile = true
+      instance_profile_name   = "centralized_vmseries_profile"
+      role_name               = "vmseries_role"
+      create_vmseries_policy  = true
+    }
+  }
+}
+
```

Comment:
Can we use optional values to specify all options available in the module here? User will know directly which parameters can be specified in the iam_policies variable